task1.js

```diff
@@ -0,0 +1,69 @@
+const choices=['Rock','Paper','Scissor'];
```

Comment:
Give proper spacing while assignment:
const choices = ['Rock','Paper','Scissor'];
update it for all cases in the code.

Reply:
Okay sir